examples/dumpers.R

```diff
@@ -0,0 +1,38 @@
+\dontrun{
```

Comment:
And one more thing. You can include this roxygen block in this file in another file by using `@template` keyword